ui/README.md

```diff
@@ -123,7 +123,7 @@ You will be taken directly to the query page. Here is where you can build your q
 
 
 ##### Build and execute a query to find all locations within the South West region:
-- click on the 'Get Elements' row
+- select 'Get Elements' from the operation drop down

```

Comment:
There are a few lines that refer to configuring the 'view'. These should be removed or changed to refer to adding filters as the View card no longer exists. Also, if a user doesn't select any groups, then all groups are returned. To simplify this walkthrough the first couple of queries could be done without specifying any groups.